tools/issue-labeler/src/CreateMikLabelModel/CreateMikLabelModel.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\eng\Versions.props" />
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
```

Comment:
Not sure that we need a Functions attribute here.  This is just a class library.